js/views/media_overlay_player.js

```diff
@@ -3,101 +3,132 @@
 //  Created by Boris Schneiderman.
 // Modified by Daniel Weck
 //  Copyright (c) 2014 Readium Foundation and/or its licensees. All rights reserved.
-//  
```

Comment:
Code auto-format should not be committed in this PR (if there is a significant and PR-relevant code change, we need to narrow it down to a palatable number of lines).